### PR TITLE
Development

### PR DESCRIPTION
**Pull request by DerekNgan**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,261 @@
...
+    border-radius: 10px;
+    box-shadow: 0 5px 5px rgba(0,0,0,0.5);
+    padding: 0 1.5rem;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    box-sizing: border-box;
+}
+
+.display h2{
```
Comment (tywy87) [inline src/App.css:99]: Desktop screen size content overflows out of container.

Screen Shot 2023-03-29 at 2 48 19 PM

File: src/App.js [hunk 2]
```diff
@@ -1,24 +1,13 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Pokedex />
```
Comment (WattBowers) [inline src/App.js:7]: You dont need the <> </> because you are only returning one thing

File: src/App.css [hunk 3]
```diff
@@ -1,38 +1,261 @@
...
-  pointer-events: none;
+.left-content,
+.right-content{
+    flex-basis: 50%;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+.left-content{
```
Comment (WattBowers) [inline src/App.css:56]: Its not 100% clear that you can click on the boxes on the left, maybe changing the cursor to pointer so the use knows that they can click on them!

File: src/Components/PokeInfo.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+                {/* // If there isn't data set it to an empty string
+                // All relevant pokemon info to be mapped through and displayed here in their proper elements(name, base stats, types)
+                // setup for the info, name, img, type, base stats */}
+                {
+                        (!poke) ? "" : 
+                        <>
+                        <div className="infoContainer">
+                             <h1>{poke.name}</h1>
+                             <img src={`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/versions/generation-v/black-white/animated/${poke.id}.gif`} alt="" />
```
Comment (WattBowers) [inline src/Components/PokeInfo.js]: No alt tag for the mapped img's

File: src/Components/PokeInfo.js [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+                        </div>
+                        </>
+                }
+                 
+                </>
+        )
+}
+
+
+// dont forget to export the component
```
Comment (WattBowers) [inline src/Components/PokeInfo.js]: lol

File: src/Components/Pokedex.js [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+         // a method that passes the response as an argument which fetches specific pokemon data using map by passing a parameter to make an api call then storing that data into a variable for pushing the data into the useState from the api and update the state within / return that data and sort the data based on pokemon id
+        //  then use the data state and prop it into the components
+                // after mapping, make api request function
+                // console.log(result)
+                // store the data objects within our array state by using the state to create a new empty array and then push the results data into that array
+                // define the sort function within the push into the state before we return so we can sort the pokemon based on id
+            const getPokemon=async(response)=>{
+                response.map(async(poke)=>{
+                    const result=await axios.get(poke.url)
+                    // console.log(result)
```
Comment (WattBowers) [inline src/Components/Pokedex.js]: console.log

File: src/Components/Pokedex.js [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+                    setPokedata(state=>{
+                        state=[...state,result.data]
+                        state.sort((a,b)=>a.id>b.id ? 1 : -1)
```
Comment (WattBowers) [inline src/Components/Pokedex.js:27]: This is super cool, no idea what this means. Gonna have to look it up

**Review by tywy87 — CHANGES_REQUESTED** [review]
Everything looks great and works well! The only issue I see is that pokemon names on the left overflow when the screen is roughly wider than 1250px. Other than that great job!

File: src/Components/PokeInfo.js [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+                                                {
+                                                    poke.stats.map((poke, index)=>{
+                                                        return(
+                                                        <h4 key={index}>{poke.stat.name}:{poke.base_stat}</h4>
+                                                        )
+                                                    })
+                                                }
+                                                
+                                        </div>
+                        </div>
```
Comment (WattBowers) [inline src/Components/PokeInfo.js]: I would maybe look at cleaning up where the div tags are, just from reading its a little unclear where the blocks start and end.

**Review by WattBowers — COMMENTED** [review]
This is super cool, love the implementation. Only bog thing I can see is just cleaning up some of the code structure, and some consol.logs hanging around!

**Pull request merged**